DESCRIPTION

```diff
@@ -64,7 +64,8 @@ Imports:
     rmarkdown (>= 2.4),
     xfun (>= 0.13),
     tinytex (>= 0.12),
-    yaml (>= 2.1.19)
+    yaml (>= 2.1.19),
+    xfun (>= 0.18)
```

Comment:
There is already a `xfun (>= 0.13)` above.

Reply:
Yes I have seen that too late, after merging, it is fixed in 0397b3042ca9229addd9cd093f33492e3579d330
Thank you.